src/checkqueue.h

```diff
@@ -81,7 +81,7 @@ class CCheckQueue
                     fAllOk &= fOk;
                     nTodo -= nNow;
                     if (nTodo == 0 && !fMaster)
-                        // We processed the last element; inform the master he can exit and return the result
+                        // We processed the last element; inform the master he or she can exit and return the result
```

Comment:
The master is not a person in this case, and again should be 'it'. Let's not antromorphize data structures.